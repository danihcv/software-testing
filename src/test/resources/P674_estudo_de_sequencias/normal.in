7
1
5
20
1 2 4 8 16 32 64 65 66 67 68 69 70 71 72 73 74 75 80 85
10
1 2 3 4 5 6 7 8 9 10
7
1 1 1 1 2 3 4
7
1 2 3 4 6 8 10
9
10 9 8 7 6 5 4 3 2
10
-10 -8 -6 -4 8 5 4 12 20 28
