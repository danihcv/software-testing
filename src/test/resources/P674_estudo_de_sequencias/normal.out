1 1
7 18
1 10
1 4
1 4
1 9
1 4
